Assets/PatchKit Patcher/Scripts/AppData/Remote/Downloaders/TorrentDownloader.cs

```diff
@@ -62,6 +62,9 @@ public void Download(CancellationToken cancellationToken)
 
                 using (var tempDir = new TemporaryDirectory(DestinationDirectoryPath))
                 {
+                    _logger.LogTrace("tempDir = " + tempDir.Path);
+                    Directory.CreateDirectory(tempDir.Path);
```

Comment:
Why TemporaryDirectory is not trying to create a directory, but it's trying to delete it?

Reply:
Done.